server/src/main/scala/actors/daily/TerminalDayFlightActor.scala

```diff
@@ -78,11 +81,33 @@ class TerminalDayFlightActor(
     restorer.finish()
   }
 
+  def getRedListCount(redListPassengers: RedListPassengers, now: () => SDateLike): Set[String] = {
+    val neboArrivalActor: ActorRef = context.system.actorOf(NeboArrivalActor.props(redListPassengers, now))
+    val neboArrivals: NeboArrivals = Await.result(neboArrivalActor.ask(GetState)(Timeout(60 seconds)).mapTo[NeboArrivals], 5 seconds)
+    neboArrivals.arrivalRedListPassengers.getOrElse(NeboArrivalActor.getRedListPassengerFlightKey(redListPassengers), Set.empty)
+  }
+
+  private def matchesScheduledAndVoyageNumber(fws: ApiFlightWithSplits, scheduled: SDateLike, voyageNumber: VoyageNumberLike) = {
```

Comment:
Add a return type for this function

Reply:
Can we specify a return type on this function please - it helps the reader determine what the function does